src/Algebra/Module/Morphism/LeftSemimoduleMonomorphism.agda

```diff
@@ -0,0 +1,138 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Consequences of a monomorphism between left semimodules
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Algebra.Module.Bundles.Raw
+open import Algebra.Module.Morphism.Structures
+
+module Algebra.Module.Morphism.LeftSemimoduleMonomorphism
+  {r a b ℓ₁ ℓ₂} {R : Set r} {M₁ : RawLeftSemimodule R a ℓ₁} {M₂ : RawLeftSemimodule R b ℓ₂} {⟦_⟧}
+  (isLeftSemimoduleMonomorphism : IsLeftSemimoduleMonomorphism M₁ M₂ ⟦_⟧)
+  where
+
+open IsLeftSemimoduleMonomorphism isLeftSemimoduleMonomorphism
+open RawLeftSemimodule M₁ renaming (Carrierᴹ to A; _≈ᴹ_ to _≈ᴹ₁_; _+ᴹ_ to _+ᴹ_; 0ᴹ to 0ᴹ₁; _*ₗ_ to _*ₗ_)
```

Comment:
Have you considered name these `M` and `N` instead, and then using qualified names on local modules instead of all these renames?

I've experimented with both styles myself, and rather grew to prefer prefixes over rename+subscripts. But it's definitely a style issue!

Reply:
Two things about this:
* I agree with @JacquesCarette and @MatthewDaggitt in their advocacy for private modules and qualified names, over renaming; the additions in [`FreeMagma`](https://github.com/agda/agda-stdlib/pull/1962) benefited enormously from this nudge;
* the existing boilerplate in `Algebra.Morphism.Structures` unfortunately adopts the 'older' style, so any attempt to migrate ought to be accompanied by a global (and exhaustingly boring) refactoring...

So, I'd be tempted to let this stand as it is for now, provided we post a `style-guide` issue on the need to do this eventually?